service/src/main/kotlin/no/nav/su/se/bakover/service/søknad/lukk/LukkSøknadServiceImpl.kt

```diff
@@ -94,16 +94,15 @@ internal class LukkSøknadServiceImpl(
                                 log.warn("Kunne ikke lukke oppgave ${søknad.oppgaveId} for søknad ${søknad.id}")
                             }
                         val sak = hentSak(lukket.sakId)
+
+                        val event = if (lukketSøknadbehandling == null) {
+                            Event.Statistikk.SøknadStatistikk.SøknadLukket(søknad = lukket, saksnummer = sak.saksnummer)
```

Comment:
Burde ikke denne eventen fyres av i begge tilfeller?

Jeg litt usikker på hvordan den statistikktjenesten aggregerer data. Men jeg tenker jo på dette som 2 forskjellige hendelser som kan oppstå parallellt. Det kan tenkes at vi i fremtiden ønsker å avslutte en behandling, men ikke lukke søknaden. Typisk "Ooops, det var ikke _denne_ søknaden jeg skulle starte en behandling for nå). For at vi deretter at søknaden blir lukket i en handling noe senere.

Reply:
Christian spurte statistikk på slack: 
![image](https://user-images.githubusercontent.com/6595794/146006556-522125c7-3cf2-4101-9f8f-8af9c86224ae.png)

Så tolker det som at det ikke trengs to. Enig? :D

Reply:
De har i praksis ikke et begrep om en søknad, det er det bare vi som har – de har kun *behandling* som konsept, hvor innkommende søknad er (eller kan være) første steg i en behandling. Så det vi sier med de to eventene er egentlig i begge tilfeller bare "avlsutt behandling".

Tipper vi må ta det scenariet ditt hvis/når det en gang kommer, for det tror jeg ihvertfall ikke de har støtte for. Mulig vi i det tilfellet *ikke* skal sende en "avslutt behandling"..

Reply:
Litt forstyrrende små de brillene hans på det profilbildet 😄  😎

Reply:
Takk for utfyllende svar! :)